File: test0/test1.java
import java.util.Scanner;
import java.text.DecimalFormat;

public class test1 {
    public static void main(String[] args){
        Scanner reader=new Scanner(System.in);
        int M=reader.nextInt();
        int N=reader.nextInt();
        int K=reader.nextInt();
        //分配空间
        double A[][]=new double[M][N];
        double B[][]=new double[N][K];
        double C[][]=new double[M][K];
        //初始化
        for(int i=0;i<M;i++){
            for(int j=0;j<N;j++){
                double num1=Math.random();
                A[i][j]=num1;
            }
        }

        for(int i=0;i<N;i++){
            for(int j=0;j<K;j++){
                double num2=Math.random();
                B[i][j]=num2;
            }
        }

        for(int i=0;i<M;i++){
            for(int j=0;j<K;j++){
                C[i][j]=0.0;
            }
        }
        //开始时间
        double startTime=System.nanoTime();
        //矩阵运算
        for(int i=0;i<M;i++){
            for(int j=0;j<K;j++){
                for(int l=0;l<N;l++){
                    C[i][j]+=A[i][l]*B[l][j];
                }
            }
        }
        //结束时间
        double endTime=System.nanoTime();
        double Usedtime=endTime-startTime;

        DecimalFormat df=new DecimalFormat("#.######");
        System.out.println("Used time: "+df.format(Usedtime/1e9)+" s");
    }
}

File: test0/test1.py
import time
import random
M,N,K=map(int,input().split())
#初始化
A=[[random.uniform(0,1) for j in range(N)] for i in range(M)]
B=[[random.uniform(0,1) for j in range(K)] for i in range(N)]
C=[[0 for j in range(K)] for i in range(M)]
#开始时间
time1=time.time()
#矩阵运算
for i in range(M):
    for l in range(N):
        for j in range(K):
            C[i][j]+=A[i][l]*B[l][j]
#结束时间
time2=time.time()
print("Used time:",(time2-time1).__round__(6),"s")